Sources/NIO/BaseStreamSocketChannel.swift

```diff
@@ -161,6 +164,7 @@ class BaseStreamSocketChannel<Socket: SocketProtocol>: BaseSocketChannel<Socket>
                     promise?.fail(ChannelError.outputClosed)
                     return
                 }
+                _close0Cleanup(mode:mode)
```

Comment:
Some minor style nits here: we'd normally want this to be:

```suggestion
                self._close0Cleanup(mode: mode)
```

The same applies to the two other instances below.

Reply:
I've fixed the callsites, thanks.